File: app/src/main/kotlin/de/ae/formulaecalendar/remote/pojo/calendar/RaceCalendar.java
package de.ae.formulaecalendar.remote.pojo.calendar;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class RaceCalendar {

    @SerializedName("RaceCalendarData")
    @Expose
    private RaceCalendarData raceCalendarData;

    public RaceCalendarData getRaceCalendarData() {
        return raceCalendarData;
    }

    public void setRaceCalendarData(RaceCalendarData raceCalendarData) {
        this.raceCalendarData = raceCalendarData;
    }

    @Override
    public String toString() {
        return "RaceCalendar{" +
                "raceCalendarData=" + raceCalendarData +
                '}';
    }
}

File: app/src/main/kotlin/de/ae/formulaecalendar/remote/pojo/calendar/RaceCalendar.kt
package de.ae.formulaecalendar.remote.pojo.calendar

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class RaceCalendar {

    @SerializedName("RaceCalendarData")
    @Expose
    var raceCalendarData: RaceCalendarData? = null

    override fun toString(): String {
        return "RaceCalendar{" +
                "raceCalendarData=" + raceCalendarData +
                '}'
    }
}
